Utils/HandlerUtil.py

```diff
@@ -61,26 +61,62 @@
 import base64
 import json
 import time
+
+from xml.etree import ElementTree
 from os.path import join
 from Utils.WAAgentUtil import waagent
 from waagent import LoggerInit
 
 DateTimeFormat = "%Y-%m-%dT%H:%M:%SZ"
 
+MANIFEST_XML = "manifest.xml"
+
 class HandlerContext:
     def __init__(self,name):
         self._name = name
         self._version = '0.0'
         return
 
 class HandlerUtility:
-    def __init__(self, log, error, short_name):
+    def __init__(self, log, error, s_name=None, l_name=None, version=None):
```

Comment:
naming nit: `version` is stored in `_extension_version`. it would be good to name this `extension_version` instead. not important.